packages/tailwindcss/src/compat/config/resolve-config.ts

```diff
@@ -176,6 +178,7 @@ function extractConfigs(ctx: ResolutionContext, { config, base, path }: ConfigFi
 function mergeTheme(ctx: ResolutionContext) {
   let api: PluginUtils = {
     theme: createThemeFn(ctx.design, () => ctx.theme, resolveValue),
+    colors,
```

Comment:
Moved this into a separate PR: https://github.com/tailwindlabs/tailwindcss/pull/14651

Fix is included here because otherwise one of the new tests would make the migrator crash